lib/jira/plugin.rb

```diff
@@ -37,7 +37,7 @@ class DangerJira < Plugin
     #
     # @return [void]
     #
-    def check(key: nil, url: nil, emoji: ":link:", search_title: true, search_commits: false, fail_on_warning: false, report_missing: true, skippable: true)
+    def check(key: nil, url: nil, emoji: ":link:", search_title: true, search_commits: false, fail_on_warning: false, report_missing: true, skippable: true, message_only_on_missing: false)
```

Comment:
Something like `jira_messages_on_non_issues` could be better here. Also please don't forget to add a comment above briefly describing the parameter.

Reply:
thanks for the note on the comment, I think mentioning jira in the flag name would be a bit redundant as this is how it will be used and none of the others do that:
```ruby
jira.check(
  key: ["DIG", "WAP", "CORE"],
  url: "https://youfoodz.atlassian.net/browse",
  search_title: true,
  search_commits: false,
  fail_on_warning: true,
  report_missing: true,
  skippable: true,
  messages_on_non_issues: false
)
```